Dockerfile

```diff
@@ -16,14 +16,12 @@ RUN apt-get update && \
 WORKDIR /app
 COPY . ./
 
-# debug
-RUN find
-RUN env
-
-# downloads and prepares static assets
+# downloads if needed and prepares static assets
 RUN python3 -m squad.frontend
-RUN ./manage.py collectstatic --noinput
-
+RUN ./manage.py collectstatic --noinput --verbosity 0
 
-USER www-data
+RUN useradd --create-home squad
+USER squad
+ENV SQUAD_STATIC_DIR /app/static
```

Comment:
I think moving this above line 21 would make line 21 more explicit and safer, unless I misunderstand something.

Reply:
In fact, it may be better to move it outside of /app, because dev-docker mounts over the top of it which causes the assets to be fetched twice.

Reply:
dev-docker discards everything below WORKDIR, so it will not be affected.